ui/job-view/App.jsx

```diff
@@ -228,7 +228,7 @@ class App extends React.Component {
   };
 
   setCurrentRepoTreeStatus = (status) => {
-    const link = document.head.querySelector('link[rel="shortcut icon"]');
+    const link = document.head.querySelector('link[rel="Treeherder Jobs View icon"]');
```

Comment:
Should this be changed, or added in addition to the current code? If I'm reading correctly, this code is currently updating something on Push Health, which won't happen anymore this change?